src/Okta.Sdk/Internal/PayloadHandler.cs

```diff
@@ -46,13 +45,15 @@ static PayloadHandler()
         /// <inheritdoc/>
         public virtual string GetBody(HttpRequest httpRequest)
         {
-            if (httpRequest.Payload != null && Serializer != null)
+            if (httpRequest.Payload == null || Serializer == null)
```

Comment:
In some places, I inverted the IF statements, since we had nested IFs. This simplifies the code and makes it much easier to reason about.